Gems/ROS2/Code/Include/ROS2/Frame/NamespaceConfiguration.h

```diff
@@ -58,8 +58,13 @@ namespace ROS2
         //! @param parentNamespace parent namespace.
         void SetParentNamespace(const AZStd::string& parentNamespace);
 
+        //! Initializes the namespace configuration.
+        //! This function should be called in the Init functions of frame components.
+        void Init();
+
     private:
         AZStd::string m_namespace;
+        AZStd::string m_customNamespace = "";
         AZStd::string m_parentNamespace;
```

Comment:
1. Please initialize all three values
2. Please add a comment to clearly differentiate between `m_namespace` and `m_customNamespace` as it might be misleading for people who work less often with ROS 2

Reply:
Pushed the fix.